messaging-activemq/src/main/resources/org/wildfly/extension/messaging/activemq/LocalDescriptions.properties

```diff
@@ -341,6 +341,10 @@ server.commit-prepared-transaction=Heuristically commits a prepared transaction.
 server.connection-ttl-override=If set, this will override how long (in ms) to keep a connection alive without receiving a ping.
 server.create-bindings-dir=Whether the server should create the bindings directory on start up.
 server.create-journal-dir=Whether the server should create the journal directory on start up.
+server.critical-analyzer-enabled=To enable or disable the criticial analyzer.
+server.critical-analyzer-timeout=Timeout used to do the critical analysis.
+server.critical-analyzer-check-period=Time used to check the response times (default half of critical-analyzer-timeout).
+server.critical-analyzer-policy=Should the server log, be halted or shutdown upon failures.
```

Comment:
Does this analyzer "halt" or "shutdown" the entire WildFly server, or just the messaging server?

If latter, perhaps s/the server log/the messaging server log/g